Sources/Build/BuildPlan.swift

```diff
@@ -45,8 +45,7 @@ extension BuildParameters {
 
     /// Extra flags to pass to Swift compiler.
     public var swiftCompilerFlags: [String] {
-        var flags = self.flags.cCompilerFlags.flatMap({ ["-Xcc", $0] })
```

Comment:
Note that this added command-line `cCompilerFlags` to the Swift flags but not Workspace toolchain `extraCCFlags`. Maybe the latter should be added instead, just pointing out the discrepancy.